simulation-model/simulation-model_shacl.ttl

```diff
@@ -0,0 +1,317 @@
+            sh:name "model.type"@en ;
+            sh:order 0 ;
+            sh:path simulation-model:model.type ],
+        [ skos:example "OpenMSL Project Group" ;
+            sh:datatype xsd:string ;
+            sh:description "Provider of the Model, no specific convention defined, it can be a company, a department a person or a combination of those."@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:message "Validation of model.manufacturer failed!"@en ;
+            sh:name "model.manufacturer"@en ;
+            sh:order 1 ;
+            sh:path simulation-model:model.manufacturer ],
+        [ skos:example "Sensor Family X4000" ;
+            sh:datatype xsd:string ;
+            sh:description "Model Family Name, in some case a model is a concrete instance of a more generic model belonging to a family. In this case, the family can be mentioned here."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.family failed!"@en ;
+            sh:name "model.family"@en ;
+            sh:order 2 ;
+            sh:path simulation-model:model.family ],
+        [ skos:example "X4101" ;
+            sh:datatype xsd:string ;
+            sh:description "Model Name. This can be the name of the model file or the name of the model as it is called"@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:message "Validation of model.name failed!"@en ;
+            sh:name "model.name"@en ;
+            sh:order 3 ;
+            sh:path simulation-model:model.name ],
+        [ skos:example "1.2.2" ;
+            sh:datatype xsd:string ;
+            sh:description "Model Version, no specific convention defined"@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:message "Validation of model.version failed!"@en ;
+            sh:name "model.version"@en ;
+            sh:order 4 ;
+            sh:path simulation-model:model.version ],
+        [ skos:example "2009-06-30T18:30:00+02:00" ;
+            sh:datatype xsd:dateTime ;
+            sh:description "Model Release Date in ISO8601"@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:message "Validation of model.releasedate failed!"@en ;
+            sh:name "model.releasedate"@en ;
+            sh:order 5 ;
+            sh:path simulation-model:model.releasedate ],
+        [ skos:example "In Development" ;
+            skos:example "Released" ;
+            skos:example "Decommissioned" ;
+            sh:datatype xsd:string ;
+            sh:description "The status of the model version released in the life cycle of the model."@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:message "Validation of model.releasestatus failed!"@en ;
+            sh:name "model.releasestatus"@en ;
+            sh:order 6 ;
+            sh:path simulation-model:model.releasestatus ],
+        [ skos:example "No use restrictions" ;
```

Comment:
I don't know :D @jtdemer do you have a opinion on that one?

Reply:
Discussed it with Johannes, `releasestatus` is for a market place not really necessary, since probably nobody wants to buy any model that is `decomissioned` or `in develeopment` in a market-place. @thomassedlmayer do you agree? Then I would delete this attribute.

The `releasedate` attribute will be integerated in `market-place` with a date when the model is available in the ecosystem.

Is it ok for you @thomassedlmayer?

Reply:
Unfortunaly, I forgot about the github feature to tag multiple lines, so I just referenced the highest line of the respective field description. My point was actually about use restrictions (L162 to L169).

Concerning your question regarding the release status: I'm not sure if there are scenarios where you want to share a pre-released version of a model over the marketplace. I don't know if this should/can be removed for the Gaia-X ontology.

Reply:
Ahh ok :D I quite misunderstood that. Yeah, that's a different question. @jtdemer should probably answer.

For the other attributes,  we just keep them since maybe someone wants to use that.

Reply:
Currently we defined user restrictions in the attached Licence file. Dataprovider have also the possiblity to set an policy in the dataspace. Do we need additional user restrictions?

Reply:
Ok, I think that fits the purpose. So I would think this attribute can be removed from this PR regarding the model specifics